crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -633,56 +633,54 @@ async fn list_contexts(
     }
```

Comment:
should we separate Asc, Desc like you did in experiment/list ?

Reply:
What would be the benefit? Could you share an example?

Reply:
@Datron https://github.com/juspay/superposition/pull/288#discussion_r1853885506